### ISSUE TITLE
Accessibility Issues in Login Page

### ISSUE THREAD

**Issue by ajitava-git** [qxf2/newsletter_automation#224]
The below-mentioned issues are classified as critical, serious, and moderate.

**Critical:**

"aria-hidden='true' must not be present on the document body"

'helpUrl': https://dequeuniversity.com/rules/axe/3.1/aria-hidden-body?application=axeAPI

**Serious:**

1. The contrast between foreground and background colors should meet WCAG 2 AA contrast ratio thresholds. Elements must have sufficient color contrast.'bgColor': None, 'contrastRatio': 0, 'expectedContrastRatio': '4.5:1', 'fgColor': '#212529' (for the contents)'helpUrl': ' https://dequeuniversity.com/rules/axe/3.1/color-contrast?application=axeAPI'

2. Ensures each page has at least one mechanism for a user to bypass navigation and jump straight to the content (role=main) 'helpUrl': 'https://dequeuniversity.com/rules/axe/3.1/bypass?application=axeAPI '

**Moderate:**

1. Ensure the headings are semantically correct

Qxf2 Newsletter Generator (h1)

Hello There (h3)

Article editors (h4)

How to use the newsletter? (h4)

Known issues (h4)

'helpUrl': ' https://dequeuniversity.com/rules/axe/3.1/heading-order?application=axeAPI'

2. Page doesn't have banner landmark, contentinfo landmark 'helpUrl': 'https://dequeuniversity.com/rules/axe/3.1/landmark-no-duplicate-banner?application=axeAPI '

**Comment by ajitava-git** [comment]
The accessibility tests run fine now in CI.
